src/main/java/datawave/microservice/query/messaging/rabbitmq/RabbitMQQueryResultsPublisher.java

```diff
@@ -0,0 +1,126 @@
+package datawave.microservice.query.messaging.rabbitmq;
+
+import static datawave.microservice.query.messaging.rabbitmq.RabbitMQQueryResultsManager.QUERY_RESULTS_EXCHANGE;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.core.MessageProperties;
+import org.springframework.amqp.rabbit.connection.CorrelationData;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import datawave.microservice.query.messaging.ClaimCheck;
+import datawave.microservice.query.messaging.QueryResultsPublisher;
+import datawave.microservice.query.messaging.Result;
+import datawave.microservice.query.messaging.config.MessagingProperties;
+
+class RabbitMQQueryResultsPublisher implements QueryResultsPublisher {
+    private final Logger log = LoggerFactory.getLogger(this.getClass());
+    
+    private static final long DEFAULT_MAX_MSG_SIZE = 536870912L;
```

Comment:
Is this a Rabbit default? If not where did we come up with this constant? Consider adding a comment with why this number was chosen

Reply:
I'll add a comment, but here's the reference:
https://github.com/rabbitmq/rabbitmq-common/blob/v3.8.0/include/rabbit.hrl#L238